src/serde.rs

```diff
@@ -106,6 +106,355 @@ pub mod uuid {
     }
 }
 
+#[cfg(feature = "chrono")]
```

Comment:
This logic should be tested in the same way as the `time` feature (via integration tests with a real CH server without `serde_json`)
https://github.com/ClickHouse/clickhouse-rs/blob/main/tests/it/time.rs

This unit tests are useless with more reliable integration tests.